app/Http/Controllers/Apis/Protected/Main/Factories/SponsoredProjectValidationRulesFactory.php

```diff
@@ -30,12 +30,22 @@ public static function build(array $data, $update = false){
                 'name' => 'sometimes|string',
                 'description' => 'sometimes|string',
                 'is_active' => 'sometimes|boolean',
+                'nav_bar_title' => 'sometimes|string',
+                'should_show_on_nav_bar' => 'sometimes|boolean',
+                'learn_more_link' => 'sometimes|string',
```

Comment:
should be https://laravel.com/docs/9.x/validation#rule-url

Reply:
done